devtools-ci-index.yaml

```diff
@@ -1881,6 +1885,26 @@
         - timed: '{ee_test_start_time}'
     <<: *e2e_tests_template
 
+- job-template:
+    name: 'devtools-{git_repo}-e2e-{test_suite}-{feature_level}'
+    test_url: prod-preview.openshift.io
+    test_suite: smoketest
+    cluster: pr-us-east-2a
+    zabbix_enabled: false
+    git_organization: openshiftio
+    git_repo: saas-openshiftio
+    beforeScript: |
+        # if the PR to saas-openshiftio is done immediatelly after the commit
+        # to dependency's master, it might be still deploying to prod-preview
+        echo "Sleeping for 1 min"
+        sleep 60
```

Comment:
I'm not sure this will work. Deployment of commit to master/prod-preview can take up to 40 min. 
However, I always wait for the commit to finish. I want to make sure the build is green.

Reply:
ping @ppitonak ^

Reply:
I'm sorry for late reply. Is there a reliable way how to programatically find out that it was deployed? Alternatively, it would be enough if we could get the version of now-deployed component and PR check job would just fail and print message that you need to wait.